### PR TITLE
Docs add instructions for docker image

### PR DESCRIPTION
**Pull request by Szarad**
Add instruction to build and run a Docker image, correct instructions and add requirements.txt file.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ENV APP_MODULE=app.main:app
+ENV LANGUAGETOOL_API=https://lt.api.witty.works/v2 
+COPY --chown=wittyuser:wittyuser requirements.txt requirements.txt
+ENV PATH="/home/wittyuser/.local/bin:${PATH}"
+RUN pip install -r requirements.txt --user
+RUN mkdir files
+RUN spacy download en_core_web_sm
+RUN spacy download de_core_news_sm
+COPY --chown=wittyuser:wittyuser . .
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0"]
```
Comment (lsmith77) [inline Dockerfile]: switch to gunicorn https://github.com/witty-works/nlp_api/blob/dev/.platform.app.yaml#L21

Comment (Szarad) [reply Dockerfile]: Could you tell me how the $SOCKET variable is load? Is it done in platform.sh or I miss sth in the code?

Comment (lsmith77) [reply Dockerfile]:
```
✔ ~/htdocs/witty.works/nlp_api [dev|…1⚑ 8]
09:16 $ platform ssh "printenv | grep SOCKET"
Enter a number to choose an app:
  [0] app
  [1] languagetool
 > 0

SOCKET=/run/app.sock
```

Comment (Szarad) [reply Dockerfile]: I made the mistake with branching. For the clean history I did reset.

**Pull request merged**